packages/@okta/vuepress-site/docs/guides/create-an-api-token/overview/index.md

```diff
@@ -1,9 +1,20 @@
 ---
 title: Overview
 ---
-API tokens are used to authenticate requests to the Okta APIs. API tokens inherit the API access of the user who creates them, so we recommend that you create a [service account user](https://help.okta.com/en/prod/Content/Topics/Security/Administrators.htm?cshid=Security_Administrators#Security_Administrators) with only the permission levels that you need for the token to perform the API tasks that you require.
+Okta API tokens are used to authenticate requests to Okta APIs.
 
-You can now interact with Okta APIs using scoped OAuth 2.0 access tokens for a number of Okta endpoints. Each access token enables the bearer to perform specific actions on specific Okta endpoints, with that ability controlled by which scopes the access token contains. For more information, see the [OAuth for Okta](/docs/guides/implement-oauth-for-okta/) guide.
-<br>
-<br>
-<NextSectionLink/>
\ No newline at end of file
+When calling an Okta API endpoint, you need to supply a valid API token in the HTTP `Authorization` header, with a valid token specified as the header value. The value needs to be prefixed with the identifier `SSWS`, which specifies the proprietary authentication scheme used. For example:
+
+```HTTP
+Authorization: SSWS 00QCjAl4MlV-WPXM...0HmjFx-vbGua
+```
+
+#### Privilege level
+
+Different Okta API operations require different admin privilege levels. API tokens inherit the privilege level of the admin account used to create them. It is therefore good practice to create a service account to use when you create API tokens, in order to be able to assign the token the specific privilege level needed. See [Administrators](https://help.okta.com/en/prod/Content/Topics/Security/Administrators.htm?cshid=Security_Administrators#Security_Administrators) for admin account types and the specific privileges of each.
```

Comment:
second sentence suggested edit:

It is therefore good practice to create a service account to use when you create API tokens so that you can assign the token the specific privilege level needed.